test/test-error.c

```diff
@@ -48,3 +48,23 @@ TEST_IMPL(error_message) {
 
   return 0;
 }
+
+
+TEST_IMPL(sys_error) {
+
+#if defined(_WIN32)
+
+  ASSERT(uv_translate_sys_error(ERROR_NOACCESS) == UV_EACCES);
+  ASSERT(uv_translate_sys_error(WSAEADDRINUSE) == UV_EADDRINUSE);
+  ASSERT(uv_translate_sys_error(ERROR_BAD_PIPE) == UV_EPIPE);
+
+#else
+
+  ASSERT(uv_translate_sys_error(EPERM) == UV_EPERM);
```

Comment:
Can you test passing in existing `UV_` errors and 0 to verify that they come back.